rest-api/src/test/java/gov/cms/qpp/conversion/api/acceptance/ValidationApiFailureAcceptance.java

```diff
@@ -58,21 +55,6 @@ void testBadPerformanceStart(String comparison, Map<String, String> override) {
 			.forEach(verifyDetail(comparison, qrda));
 	}
 
-	private static Stream<Arguments> getFailureScenarios() {
```

Comment:
this is being used by testBadPerformanceStart method

Reply:
Yes, I noticed that when tests failed. I made the appropriate changes and forgot to commit them.